.drone.star

```diff
@@ -118,16 +120,16 @@ config = {
                 "OCIS_ADD_RUN_SERVICES": "antivirus",
             },
         },
-        "apiSearch": {
+        "apiSearch1": {
```

Comment:
only as suggestions: `apiContentSearch` or `apiFullTextSearch` and leave only `contentSearch.feature`

Reply:
I will try that. let's see how it will be in CI

Reply:
the pipeline runs for ~17 mins. I want them to be around 15, so have to find another way around

Reply:
maybe do like:  
```
apiSearchContent-ocis - 6.25
apiSearch1 - 8.5
apiSearch2 - 8.5
```

Why do we need the prefix `osis'? 

<img width="333" alt="Screenshot 2024-01-18 at 09 35 59" src="https://github.com/owncloud/ocis/assets/84779829/e5c62b05-c5a9-479c-a86a-dbc3f5c0f2d3">

Reply:
that should be the storage name. previously we used to have pipelines for different storages